app/javascript/components/modal/Modal.vue

```diff
@@ -14,7 +14,7 @@
           class="modal__close"
           @click="closeModal"
         />
-
+        <h3>Sources</h3>
```

Comment:
Can this go in the yml? You can then use the vue translation store to access the translation directly in the vue comp.

Reply:
I could, but then my concern is that I would have to add this in for every YML file (as there is a separate YML file for each modal) so I would be repeating myself.

Reply:
@stanleypliu You can put it in a global yaml file and then pull it into the component once, if that makes sense? 

See this component for an example: https://github.com/unepwcmc/ocean-plus-habitats/blob/47209bc0f95b571bdf351a12ea24919811b4ff3f/app/javascript/components/map/filters/MapDownloadButton.vue#L73-L86